packages/passgan/PKGBUILD

```
@@ -0,0 +1,45 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=passgan
+pkgver=38.5927158
+pkgrel=1
+groups=('blackarch' 'blackarch-cracker')
+pkgdesc='A Deep Learning Approach for Password Guessing.'
+url='https://github.com/D3vil0p3r/PassGAN'
+arch=('x86_64' 'aarch64')
```

```suggestion
arch=('any')
```

Use any unless it's compile stuff.